vowpalwabbit/baseline.cc

```diff
@@ -107,7 +107,7 @@ void predict_or_learn(baseline& data, single_learner& base, example& ec)
       init_global(data);
       data.global_initialized = true;
     }
-    VW::copy_example_metadata(/*audit=*/false, data.ec, &ec);
```

Comment:
whats the deal with audit being slowly removed

Reply:
Well it wasn't necessary in this function. All audit is copied automatically if it exists. It may be a leftover from how this code used to work